wordpress-mobile/gutenberg-mobile
# Image set via parent media library picker

Addresses #192 for the Android side of the Bridge.

This PR adds a function to the bridge interface to trigger the media library image picker on the parent native app. The function provides a callback for the Bridge to invoke to send the selected image URL back into RN.

Depends on the GB repo PR (needs merge first): https://github.com/WordPress/gutenberg/pull/11511

To test (Android):

1. Use the `feature/integrate-gutenberg-image-set` branch from https://github.com/wordpress-mobile/WordPress-Android. [PR here](https://github.com/wordpress-mobile/WordPress-Android/pull/8553).
2. update all submodules at every level
3. Run `yarn wpandroid` at the wpandroid root to compile the native app, run Metro, install and run the mobile app in a single command
4. Open a GB post in wpandroid and add a new image block
5. Tap on the "MEDIA LIBRARY" button and select an image from the *media library*
6. App should update the block and show the image
